config/samples/sfptpd/conf/ptp1/configmap.yaml

```diff
@@ -0,0 +1,22 @@
+# SPDX-License-Identifier: MIT
+# (c) Copyright 2023 Advanced Micro Devices, Inc.
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: sfptpd-config
+  namespace: sfptpd
+data:
+  ptp1.conf: |
+    [general]
+    sync_module ptp ptp1
+
+    # Log to stdout/stderr. The recommended way to deploy would be only logging
+    # the message log, to stderr, and placing an adapter program in the pod to
+    # process RT JSON stats and make them available to a time series database.
+    message_log stderr
+    stats_log stdout
+
+    [ptp1]
+    ptp_domain 0
+    transport ipv6
```

Comment:
As this will certainly need customising by the user I suggest adding an example interface stanza and removing the `ipv6` example which was only there previously to provide out-of-the-box action using automatic link local address but is very unlikely to match the user's requirements.